test/e2e/localnet-underlay.go

```diff
@@ -13,41 +13,56 @@ import (
 )
 
 const (
-	bridgeName = "ovsbr1"
-	add        = "add-br"
-	del        = "del-br"
+	defaultOvsBridge = "breth0"
+	secondaryBridge  = "ovsbr1"
+	add              = "add-br"
+	del              = "del-br"
 )
 
-func setupUnderlay(ovsPods []v1.Pod, portName string, nadConfig networkAttachmentConfig) error {
+func configureBridgeMappingsForNode(ovsPod v1.Pod, networkName, bridgeName string) error {
```

Comment:
I think it weird that a function named `configureBridgeMappingsForNode` does not receive the node name as a parameter. 

And I also find it weird it simply wraps another function call, omitting the fact it configures the cluster default network mapping, *plus* the mapping you want to have configured.

Reply:
tks, I'm resolving this, since it's now IMHO simpler.